UnitsNet/UnitDefinitions/FoulingFactor.json

```diff
@@ -0,0 +1,68 @@
+{
+  "Name": "FoulingFactor",
```

Comment:
I do think the name `ThermalResistance` seems better. It is more generic and not tied to heat exchangers, but to any kind of heat transfer.
What do you think?